charts/kafka-operator/crds/operator-kafka-crd.yaml

```diff
@@ -499,6 +499,47 @@ spec:
                           type: string
                       type: object
                     type: array
+                  envoy:
+                    description: Custom properties for the dedicated envoy deployment
+                      for this broker configuration group.
+                      Applicable only if EnvoyConfig.dedicatedEnvoyPerBrokerGroup is
+                      set to true.
+                    properties:
+                      replicas:
+                        format: int32
+                        type: integer
+                    type: Object
+                  listenersConfig:
+                    description: ListenersConfig defines the broker specofic Kafka
+                     listener types. These will override the global ListenersConfig
+                    properties:
+                      externalListeners:
+                        items:
+                          description: ExternalListenerConfig defines the external listener
+                            config for Kafka
+                          properties:
+                            containerPort:
+                              format: int32
+                              type: integer
+                            externalStartingPort:
+                              format: int32
+                              type: integer
+                            hostnameOverride:
+                              type: string
+                            name:
+                              type: string
+                            type:
+                              type: string
+                          required:
+                            - containerPort
```

Comment:
not clear what `containerPort` and `externalStartingPort` are ?

Reply:
`externalStartingPort` is the port that will be used at LB level, wile the `containerPort` is the port used by Envoy to access the broker